subscripts/script_1.m

```diff
@@ -1,10 +1,10 @@
 %% script_1
 
-%read in data
-data = csvread('data/peach.csv');
-%plot data
-p = patch(data(:,1), data(:,2), 'orange');
-p.LineWidth = 2;
+% %read in data
+% data = csvread('data/peach.csv');
+% %plot data
+% p = patch(data(:,1), data(:,2), 'orange');
+% p.LineWidth = 2;
 
 %% This section has the fixed code if you are struggling to debug!
```

Comment:
can you add a reference to breakout 1 in this section header so it matches what you did for breakout 2? This applies to all subscripts.